src/methods/democracy/closeProxy.ts

```diff
@@ -17,6 +17,10 @@ export function closeProxy(
   info: BaseTxInfo,
   options: OptionsWithMeta
 ): UnsignedTransaction {
+  console.warn(
```

Comment:
@amaurymartiny should these warnings be in the docs? Either way is fine with me.

Reply:
We can add a `@deprecated` in the doc comments, these functions will then show as deprecated in the generated docs.